src/it/scala/io/scalac/tezos/translator/LibrarySpec.scala

```diff
@@ -59,6 +62,14 @@ class LibrarySpec
       }
     }
 
+    private def getToken(userService: UserService, user: UserCredentials): String = {
+      val maybeToken = Await.result(userService.authenticateAndCreateToken(user.username, user.password), 3 seconds)
+
+      maybeToken shouldBe a[Some[_]]
+
+      maybeToken.get
+    }
+
```

Comment:
It's easier to just mock the "authenticateAndPrependUsername" method

Reply:
https://github.com/ScalaConsultants/Tezos-FullStack-Console-backend/pull/25#discussion_r352535446